src/marionette.rs

```diff
@@ -334,11 +334,12 @@ impl MarionetteHandler {
                          .map_err(|_| WebDriverError::new(ErrorStatus::UnknownError,
                                                           "Unable to read profile preferences file")));
 
-        prefs.insert("marionette.defaultPrefs.port", Pref::new(port as i64));
+        for &(ref name, ref value) in prefs::DEFAULT.iter() {
```

Comment:
With the customers I support, the usual pattern is that they start by using the default settings, and at some point they realize that they need to install an extension or set up some certs that can (currently) most easily be accomplished by providing a custom profile. It is then surprising when a profile created with the sole purpose of doing this causes other behavior changes DEFAULT prefs are no longer being set. My goal with the change is to make that transition from no profile to providing a profile more seamless by only changing things that the user has explicitly chosen to change when providing the profile.